src/onepay-checkout.js

```diff
@@ -1,5 +1,7 @@
+import {styles} from 'checkout.css';
```

Comment:
Out of curiosity: This applies the CSS to the current page? Or you have to manually use the `styles` variable later on?

Reply:
This apply the CSS to page that use the script